pkg/commands/product/botmanagement/common.go

```diff
@@ -0,0 +1,23 @@
+package botmanagement
+
+import (
+	"github.com/fastly/go-fastly/v9/fastly"
+	"github.com/fastly/go-fastly/v9/fastly/products/botmanagement"
+
+	"github.com/fastly/cli/internal/productcore"
+	"github.com/fastly/cli/pkg/api"
+)
+
+// EnablementHooks is a structure of dependency-injection points used
+// by unit tests to provide mock behaviors
+var EnablementHooks = productcore.EnablementHookFuncs[*botmanagement.EnableOutput]{
```

Comment:
The `*botmanagement.EnableOutput` here confused me at first, and I was wondering why there were all these different types when everything here is nicely generic, but then I realized it is the `go-fastly` API response. Not certain this calls for an import alias, but I would find it reasonable to use one here.

Reply:
Yeah... I thought about using an import alias here, as I had to do search-and-replace in 9 sets of source files :-)